### PR TITLE
readme: update 

### PR DESCRIPTION
**Pull request by tupakkatapa**
~~Reverting the regressions introduced in [this commit](https://github.com/ponkila/HomestakerOS/commit/e220ec447622167a79ed3135c0b91a31eb80e9f8). This doesn't mean that this is the final solution, but it's nice to have a working main branch.~~ Also, updating the README to be more informative, and possibly updating the documentation as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,80 @@
...
+
+## 🔍 Looking Ahead
+
+For those interested in the details, let's dive into the documentation from here.
+
+- [1. 📚 Getting Started](./docs/getting_started.md)
+
+  Learn how to set up your machine for deploying HomestakerOS, including essential setup and configurations.
+
+- [2. 🤔 What is Ethereum](./docs/what_is_ethereum.md)
```
Comment (jhvst) [inline README.md:65]: Could be called "tutorial for homestaking" instead.

File: README.md [hunk 2]
```diff
@@ -1,35 +1,80 @@
-# HomestakerOS
+# 🚀 HomestakerOS
```
Comment (jhvst) [inline README.md:1]: emoji could be Home + upwards stock chart instead

File: README.md [hunk 3]
```diff
@@ -1,35 +1,80 @@
...
 
-The wizard produces Linux disk images based on NixOS. NixOS allows configurations to be public, deterministic, and self-upgrading. Further, by loading the whole operating system into the RAM, we can eliminate the works on my machine tantrum, while also making it possible to be booted by double-clicking a kernel execution script -- and if you want to return to your previous distribution, just restart your computer.
+This is still a work in progress. We are actively seeking funding and support for ongoing development!
 
-Check out the live demo at https://demo.homestakeros.com/
+## 📋 Overview
 
-## How to Run
+You can configure and build your host with a simple form and push of a button, without requiring extensive knowledge of the Nix language. The resulting images are deployed by loading the entire operating system into RAM. This makes it possible to deploy the OS in various ways, such as netbooting, or even booting by double-clicking a kernel execution script. If you want to return to your previous distribution, just restart your computer!
```
Comment (jhvst) [inline README.md:9]: Nix should be introduced first

File: README.md [hunk 4]
```diff
@@ -1,35 +1,80 @@
-# HomestakerOS
+# 🚀 HomestakerOS
 
-HomestakerOS is a web UI which creates custom Linux OS for Ethereum homestaking. It aims to democratize homestaking by simplifying the process of creating and maintaining servers in home environments.
+HomestakerOS is a web UI that creates a custom Linux OS for Ethereum homestaking. It aims to democratize homestaking by simplifying the process of creating and maintaining servers in home environments.
 
-The wizard produces Linux disk images based on NixOS. NixOS allows configurations to be public, deterministic, and self-upgrading. Further, by loading the whole operating system into the RAM, we can eliminate the works on my machine tantrum, while also making it possible to be booted by double-clicking a kernel execution script -- and if you want to return to your previous distribution, just restart your computer.
+This is still a work in progress. We are actively seeking funding and support for ongoing development!
```
Comment (jhvst) [inline README.md:6]: a screenshot and url to demo here

File: README.md [hunk 5]
```diff
@@ -1,35 +1,80 @@
...
+
+- [1. 📚 Getting Started](./docs/getting_started.md)
+
+  Learn how to set up your machine for deploying HomestakerOS, including essential setup and configurations.
+
+- [2. 🤔 What is Ethereum](./docs/what_is_ethereum.md)
+
+  A deep dive into Ethereum’s architecture, from the execution and consensus layers to running Ethereum nodes and clients.
+
+- [3. ⚙️ Client Configuration](./docs/client_configuration.md)
```
Comment (jhvst) [inline README.md:69]: should probably be merged with chapter 2

File: README.md [hunk 6]
```diff
@@ -1,35 +1,80 @@
...
+
+  A guide to configuring popular Ethereum clients, including execution and consensus options.
+
+- [4. 📖 Details](./docs/workflow.md)
```
Comment (jhvst) [inline README.md:73]: could be more like "working with Nix"

File: README.md (hunk 4 above)
Comment (tupakkatapa) [reply README.md:6]: Done. I think the screenshot would be too much, since the link is right there

**Pull request merged**